docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: "3.8"
+
+services:
+  server:
+    build: .
+    environment:
```

I suggest adding an env var to override the group ID. This helps us Linux folks who have to deal with stricter file permissions.

```suggestion
    user: :${DAEMON_GROUP_ID:-1420}
    environment:
```